spec/Classes.tex

```diff
@@ -888,10 +888,10 @@ \section{The {\em this} Method}
 \index{this@\chpl{this}}
 \index{classes!this@\chpl{this}}
 
-A procedure method declared with the name \chpl{this} allows a class to be
-``indexed'' similarly to how an array is indexed.  Indexing into a
-class instance has the semantics of calling a method
-named \chpl{this}.  There is no other way to call a method
+Declaring a procedure method with the name \chpl{this} allows the
+class's objects to be ``called'' or ``indexed,'' similarly to how
+arrays are indexed.  Such expressions have the semantics of calling
+the method named \chpl{this}.  There is no other way to call a method
```

Comment:
"Such expressions" is unclear to me. How about "A calling or indexing expression has the semantics of ..."
I would add something like "The call arguments or indices in such an expression are interpreted as the arguments to the call to \chpl{this}."